opentech/apply/funds/models/submissions.py

```diff
@@ -677,6 +677,11 @@ def get_absolute_url(self):
         })
 
 
+class AssignedReviewersQuerySet(models.QuerySet):
+    def role_reviewers_by_submission(self, submission):
```

Comment:
This should probably be `with_roles` and then we can do `self.instance.assigned.with_roles().values('reviewers')` `assigned` will already be limited to the submission